components/stripe/stripe.app.js

```diff
@@ -2,15 +2,19 @@
 
 const stripe = require("stripe");
 
-const createOptionsMethod = (collectionOrFn, keysOrFn) => async function ({ prevContext }) {
+const createOptionsMethod = (collectionOrFn, keysOrFn) => async function ({
+  prevContext, ...opts
+}) {
+  const { nextPageToken: pageToken } = prevContext;
```

Comment:
I would set a variable name like
```suggestion
  const { starting_after } = prevContext;
```